src/test/ui/rfcs/rfc-2396-target_feature-safe-calls.rs

```diff
@@ -0,0 +1,19 @@
+#[target_feature(enable = "sse2")]
```

Comment:
Seems like this isn't feature gated. Let's fix that and add a feature gate test in another file.